terraform/modules/apps/content-store/main.tf

```
@@ -78,7 +78,7 @@ module "app" {
           "awslogs-create-group" : "true",
           "awslogs-group" : "awslogs-fargate",
           "awslogs-region" : "eu-west-1", # TODO: hardcoded region
-          "awslogs-stream-prefix" : "awslogs-content-store"
+          "awslogs-stream-prefix" : "awslogs-draft-${var.service_name}",
```

There's a stray `draft-` here
```suggestion
          "awslogs-stream-prefix" : "awslogs-${var.service_name}",
```

---

good catch